c++/src/Timezone.cc

```diff
@@ -659,7 +659,15 @@ namespace orc {
   const char* getTimezoneDirectory() {
     const char* dir = getenv("TZDIR");
     if (!dir) {
-      dir = DEFAULT_TZDIR;
+      // this is present if we're in an activated conda environment
+      const char* conda_prefix = getenv("CONDA_PREFIX");
```

Comment:
Usually we use variable naming style like `condaPrefix`

Reply:
sure!